pydantic_odm/encoders/mongodb.py

```diff
@@ -42,20 +40,52 @@ def _convert_enums(
     # Iterate passed data
     for key, value in iterator:
         # Replace enum object to enum value
```

Comment:
Replace comment to:
```python
# Convert and replace value with transform function
```

Reply:
Fixed in dada6db94ba58c708033c7180b5e58023323284e